Missionframework/modules/00_init/fnc/fn_init_settings.sqf

```diff
@@ -0,0 +1,396 @@
+    localize "STR_KPLIB_SETTINGS_GENERAL",
+    true,
+    1,
+    {}
+] call CBA_Settings_fnc_init;
+
+// KPLIB_param_fobRange
+// Build radius in meters around the FOB area center position.
+// Default: 125 meters
+[
+    "KPLIB_param_fobRange",
+    "SLIDER",
+    [localize "STR_KPLIB_SETTINGS_GENERAL_FOBRANGE", localize "STR_KPLIB_SETTINGS_GENERAL_FOBRANGE_TT"],
+    localize "STR_KPLIB_SETTINGS_GENERAL",
+    [100, 250, 125, 0],
+    1,
+    {}
+] call CBA_Settings_fnc_init;
+
+// KPLIB_param_sectorActRange
+// Radius in meters around the sector center to activate the sector.
+// Default: 1200 meters
+[
+    "KPLIB_param_sectorActRange",
+    "SLIDER",
+    [localize "STR_KPLIB_SETTINGS_GENERAL_SECACT", localize "STR_KPLIB_SETTINGS_GENERAL_SECACT_TT"],
+    localize "STR_KPLIB_SETTINGS_GENERAL",
+    [600, 1800, 1200, 0],
+    1,
+    {}
+] call CBA_Settings_fnc_init;
+
+// KPLIB_param_sectorCapRange
+// Radius in meters around the sector center a unit has to be to being able to capture the sector.
+// Default: 150 meters
+[
+    "KPLIB_param_sectorCapRange",
+    "SLIDER",
+    [localize "STR_KPLIB_SETTINGS_GENERAL_SECRANGE", localize "STR_KPLIB_SETTINGS_GENERAL_SECRANGE_TT"],
+    localize "STR_KPLIB_SETTINGS_GENERAL",
+    [100, 200, 150, 0],
+    1,
+    {}
+] call CBA_Settings_fnc_init;
+
+/*
+    ----- BI REVIVE SETTINGS -----
+    Ref: https://community.bistudio.com/wiki/Arma_3_Revive
+*/
+
+// KPLIB_param_reviveMode
+// Enables/Disables the BI Revive System. (Automatically disabled if using ACE medical)
+// Default: Enabled
+[
+    "KPLIB_param_reviveMode",
+    "LIST",
+    [localize "STR_A3_ReviveMode", localize "STR_KPLIB_SETTINGS_REVIVE_MODE_TT"],
+    localize "STR_KPLIB_SETTINGS_REVIVE",
+    [
+        [
+            0,
+            1
+        ],
+        [
+            localize "STR_A3_Disabled",
+            localize "STR_A3_EnabledForAllPlayers"
+        ],
+        1
+    ],
+    1,
+    {}
+] call CBA_Settings_fnc_init;
+
+// KPLIB_param_reviveDuration
+// How long it takes to revive someone in seconds.
+// Default: 6
+[
+    "KPLIB_param_reviveDuration",
+    "LIST",
+    [localize "STR_A3_ReviveDuration", localize "STR_KPLIB_SETTINGS_REVIVE_DURATION_TT"],
+    localize "STR_KPLIB_SETTINGS_REVIVE",
+    [[6, 8, 10, 12, 15, 20, 25, 30], ["6", "8", "10", "12", "15", "20", "25", "30"], 0],
+    1,
+    {}
+] call CBA_Settings_fnc_init;
+
+// KPLIB_param_reviveRequiredTrait
+// Controls whether special training is required to revive someone.
+// Default: 1
+[
+    "KPLIB_param_reviveRequiredTrait",
+    "LIST",
+    [localize "STR_A3_RequiredTrait", localize "STR_KPLIB_SETTINGS_REVIVE_REQTRAIT_TT"],
+    localize "STR_KPLIB_SETTINGS_REVIVE",
+    [
+        [
+            0,
+            1
+        ],
+        [
+            localize "STR_A3_None",
+            localize "STR_A3_Medic"
+        ],
+        1
+    ],
+    1,
+    {}
+] call CBA_Settings_fnc_init;
```

Comment:
BIS revive is also initialized in core.

Reply:
That's true, of course. The BI Revive category will be moved to core then. 👍